mdsuite/calculators/calculator.py

```diff
@@ -452,11 +482,14 @@ def _save_data(self, name: str, data: pd.DataFrame):
                 db[self.database_group].create_dataset(title, data=data, dtype=float)
         """
 
-    def _plot_fig(self, fig: matplotlib.figure.Figure, ax: Axes,
+    def _plot_fig(self,
+                  fig: matplotlib.figure.Figure,
+                  ax: Axes,
```

Comment:
Why import `Axes` but not `Figure` ?

Reply:
I didn't add this, i just reformatted it.